app/src/main/java/vn/sunasterisk/music_67/ui/genres/GenreActivity.kt

```diff
@@ -70,6 +73,12 @@ class GenreActivity : BaseActivity(), GenresContract.View, View.OnClickListener
 				PlayingSet.playingSet.addAll(genrePresent.tracks)
 				firstTrackClicked()
 			}
+			imageSearch -> {
+				val searchIntent = Intent(this, SearchActivity::class.java).apply {
+					action = ACTION_SEARCH_FROM_HOME
+				}
+				startActivity(searchIntent)
```

Comment:
Quỳnh Lâm có thể khai báo thức này ở SeachActivity và sau đó ở đây chỉ cần gọi ra Intent
tương tự ở nơi khác
```suggestion
companion object {
        fun getIntent(context: Context) = Intent(context, SearchActivity::class.java)
    }
```